src/main/java/baritone/pathing/movement/movements/MovementDescend.java

```diff
@@ -147,9 +147,9 @@ public static boolean dynamicFallCost(CalculationContext context, int x, int y,
         int effectiveStartHeight = y;
         for (int fallHeight = 3; true; fallHeight++) {
             int newY = y - fallHeight;
-            if (newY < 0) {
+            if (newY < context.getBaritone().getPlayerContext().world().dimensionType().minY()) {
```

Comment:
Better use `context.world.getMinBuildHeight()`.
There is kind of a split between things that happen on the pathing thread (`CalculationContext`) and things that happen on the main thread (`IPlayerContext`) so using the `IPlayerContext` on the pathing thread is not nice.